app/assets/javascripts/components/locales/fr.jsx

```diff
@@ -67,9 +67,9 @@ const fr = {
   "navigation_bar.edit_profile": "Modifier le profil",
   "navigation_bar.favourites": "Favoris",
   "navigation_bar.follow_requests": "Demandes de suivi",
-  "navigation_bar.info": "Plus d'informations",
+  "navigation_bar.info": "Plus d’informations",
   "navigation_bar.logout": "Déconnexion",
-  "navigation_bar.mutes": "Comptes muets",
+  "navigation_bar.mutes": "Comptes silencés",
```

Comment:
I believe "silenciés" would be more natural - at least, I hear it more than "silencé" in casual speach